include/deal.II/base/quadrature_point_data.h

```diff
@@ -328,7 +328,7 @@ namespace parallel
     {
     public:
       DEAL_II_STATIC_ASSERT(std::is_base_of<TransferableQuadraturePointData, DataType>::value,
-                            "User's DataType class should be derived from QPData");
+                            "User's DataType class should be derived from QPData")
```

Comment:
IIRC correctly that is the only way to handle semicolons correctly in this case.